ts/boot/configureStoreAndPersistor.ts

```diff
@@ -25,6 +25,7 @@ import {
 } from "../store/reducers";
 import { getInitialState as getInstallationInitialState } from "../store/reducers/notifications/installation";
 import { GlobalState, PersistedGlobalState } from "../store/reducers/types";
+import { PotTransform } from "../store/transforms/potTransforms";
```

Comment:
this change shouldn't be in this PR - @fpersico ?

Reply:
Don't know why it is here. I think @Undermaken is testing if this PR fixes the errors in the other story?

Reply:
I don't know too. I created this branch starting from master and I can't figure how these changes are included.